hc/.gitignore

```diff
@@ -0,0 +1,103 @@
+# Byte-compiled / optimized / DLL files
```

Comment:
Why did you have to place the `.gitignore` under the `hc/` directory??
I guess it is supposed to be kept in the root folder.

Reply:
My bad, I may have dragged the file into `hc/` unknowingly. Will sort it out

Reply:
Not yet sorted